tests/Mailinglist/SubscribeUserToMailinglistRequestHandlerTest.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Mailinglist;
+
+use CultuurNet\UDB3\Http\ApiProblem\ApiProblem;
+use CultuurNet\UDB3\Http\ApiProblem\AssertApiProblemTrait;
+use CultuurNet\UDB3\Http\Request\Psr7RequestBuilder;
+use CultuurNet\UDB3\Mailinglist\Client\MailinglistClient;
+use CultuurNet\UDB3\Model\ValueObject\Web\EmailAddress;
+use Fig\Http\Message\StatusCodeInterface;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Log\LoggerInterface;
+
+class SubscribeUserToMailinglistRequestHandlerTest extends TestCase
+{
+    use AssertApiProblemTrait;
+
+    private const EMAIL = 'test@publiq.be';
+    private const MAILINGLIST_ID = 1;
+
+    private SubscribeUserToMailinglistRequestHandler $handler;
+
+    /**
+     * @var MockObject|LoggerInterface
+     */
+    private $mailinglistClient;
+
+    /**
+     * @var MockObject|LoggerInterface
+     */
+    private $logger;
+
+    private ServerRequestInterface $request;
+
+    protected function setUp(): void
```

Comment:
I think this misses a test for when the MailinglistClient would throw an exception